tests/e2e/Services/Functions/FunctionsCustomServerTest.php

```diff
@@ -1765,6 +1801,24 @@ public function testFunctionsDomainBinaryRequest()
             'execute' => ['any']
         ]);
 
+        $rule = $this->client->call(
```

Comment:
Notice below this addition, we do call to listRUles, to get $domain.
Since we are now the one making domain, we dont need that below call anymore.
Let's fix this in all places where we did this change, in this test

Reply:
I added `getFunctionDomain` just like `getSiteDomain` instead

Reply:
Double chek in all places where added createRule